go.mod

```diff
@@ -5,4 +5,5 @@ go 1.14
 require (
 	github.com/howeyc/crc16 v0.0.0-20171223171357-2b2a61e366a6
 	github.com/jbuchbinder/gopnm v0.0.0-20150223212718-5176c556b9ce
+	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
```

Comment:
golint is a tool? that shouldn't be in the app?
```suggestion
```

Reply:
I figure that's the `// indirect` bit that it is a dev-dependency, these were added automatically by the tools in VSC.